test/MultipleRedeemers.purs

```diff
@@ -0,0 +1,61 @@
+module Test.MultipleRedeemers (suite) where
```

Comment:
This would be nice once we have Puppeteer tests, but I don't want to include more dead code than we already have

Reply:
cc @MangoIV what's the state of this module?